sample/kotlin/src/main/res/values/strings.xml

```diff
@@ -60,6 +60,7 @@
     <string name="button_async_operation">Async Operation</string>
     <string name="button_network_request">Network Request</string>
     <string name="button_start">Start</string>
+    <string name="button_start_sse">Start SSE Request</string>
```

Comment:
minor: maybe use full form? `Start Server-Side Event Request`

Reply:
I could but it'll make the button multiline on most device 😅